ui/src/shared/constants/fluxFunctions.ts

```diff
@@ -306,24 +354,25 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
         type: 'Boolean',
       },
       {
-        name: 'columns',
+        name: 'column',
         desc:
-          'A list of columns on which to operate. Defaults to `["_value"]`.',
-        type: 'Array of Strings',
+          'The column on which to operate. Defaults to `"_value"`.',
+        type: 'String',
       },
       {
         name: 'timeColumn',
         desc: 'The column name for the time values. Defaults to `"_time"`.',
         type: 'String',
       },
     ],
+    package: '',
     desc:
       'Computes the rate of change per unit of time between subsequent non-null records. The output table schema will be the same as the input table.',
     example:
-      'derivative(unit: 1s, nonNegative: true, columns: ["_value"], timeColumn: "_time")',
+      'derivative(unit: 1s, nonNegative: true, column: "_value", timeColumn: "_time")',
```

Comment:
the spec still says this is "columns": https://github.com/influxdata/flux/blob/master/docs/SPEC.md#derivative

Reply:
You're right. Got over aggressive with my aggregate updates. Will fix this.